lib/linguist/languages.yml

```diff
@@ -2668,6 +2669,16 @@ OpenSCAD:
   tm_scope: none
   ace_mode: scad
 
+OpenType Font Feature:
+  type: programming
```

Comment:
I know that PS is TC and so is TeX, but like CSS they both have `type: markup` in `languages.yml`. CSS is a declarative language (and doesn’t mark up text)), but so are functional programming languages …